testcafe/tests/pages/photos-viewer/photos-viewer-model.js

```diff
@@ -70,4 +71,39 @@ export default class PhotoViewer extends Viewer {
       //!FIXME add data-photo-id=xxx in photo and check url=#/photos/xxx
     }
   }
+
+  //@param {String} screenshotPath : path for screenshots taken in this test
+  //@param {number} startIndex : index of the 1st photos to open
+  //@param {number} numberOfNavigation : the number of file we want to go through during the test.
+  async openPhotoAndCheckViewerNavigation(
+    screenshotPath,
+    startIndex,
+    numberOfNavigation
+  ) {
+    console.log(`↳ 📁 photo with index : ${startIndex}`)
+    await this.openPhotoFullscreen(startIndex)
+    await this.navigateInViewer(screenshotPath, startIndex, numberOfNavigation)
+    await this.closeViewer({
+      exitWithEsc: false
+    })
+  }
+
+  //@param {String} screenshotPath : path for screenshots taken in this test
+  //@param {string} index : file to check
+  async openPhotoAndCheckViewer(screenshotPath, index, hasMask = false) {
```

Comment:
Nit: It's more logical to group related parameters together, so the order woud make more sense as `index, screenshotPath, hasMask = false` for example.

Reply:
OK, parameters changes for 
```openPhotoAndCheckViewerNavigation(
    startIndex,
    numberOfNavigation,
    screenshotPath)```
and  `openPhotoAndCheckViewer(index, screenshotPath, hasMask = false)`